### PR TITLE
Issue #138 Update metadata structure for v1 metadata json.

### PR DESCRIPTION
**Pull request by praveenkumar**
Add the dns entry for `api-int` in the libvirt network.
Update the constants according to new domain name.

I shared the updated tarball over email notification.

### CODE REVIEW

File: pkg/crc/constants/constants.go [hunk 1]
```diff
@@ -16,9 +16,9 @@ const (
 
 	CrcEnvPrefix = "CRC"
 
-	DefaultBundle        = "crc_libvirt_v4.1.0-rc0.tar.xz"
-	DefaultHostname      = "crc-jtskh-master-0"
-	DefaultWebConsoleURL = "https://console-openshift-console.apps.tt.testing"
+	DefaultBundle        = "crc_libvirt_v4.1.0-rc5.tar.xz"
+	DefaultHostname      = "crc-g8g2v-master-0"
+	DefaultWebConsoleURL = "https://console-openshift-console.apps-crc.testing"
```
Comment (anjannath) [inline pkg/crc/constants/constants.go]: :+1:

File: pkg/crc/machine/libvirt/templates.go [hunk 2]
```diff
@@ -18,12 +11,13 @@ const (
...
-	<domain name='crc.tt.testing' localOnly='yes'/>
+	<domain name='crc.testing' localOnly='yes'/>
 	<dns>
-	  <srv service='etcd-server-ssl' protocol='tcp' domain='crc.tt.testing' target='etcd-0.crc.tt.testing' port='2380' weight='10'/>
+	  <srv service='etcd-server-ssl' protocol='tcp' domain='crc.testing' target='etcd-0.crc.testing' port='2380' weight='10'/>
 	  <host ip='{{ .IP }}'>
-		<hostname>api.crc.tt.testing</hostname>
-		<hostname>etcd-0.crc.tt.testing</hostname>
+		<hostname>api.crc.testing</hostname>
+		<hostname>api-int.crc.testing</hostname>
```
Comment (anjannath) [inline pkg/crc/machine/libvirt/templates.go]: :+1:

File: pkg/crc/preflight/preflight_checks_linux.go [hunk 3]
```diff
@@ -33,8 +33,8 @@ const (
 var (
 	libvirtDriverBinaryPath = filepath.Join(driverBinaryDir, libvirtDriverCommand)
 	crcDnsmasqConfigPath    = filepath.Join(string(filepath.Separator), "etc", "NetworkManager", "dnsmasq.d", crcDnsmasqConfigFile)
-	crcDnsmasqConfig        = `server=/tt.testing/192.168.130.1
-address=/apps.tt.testing/192.168.130.11
+	crcDnsmasqConfig        = `server=/crc.testing/192.168.130.1
+address=/apps-crc.testing/192.168.130.11
```
Comment (anjannath) [inline pkg/crc/preflight/preflight_checks_linux.go]: :+1:

**Comment by tsedmik** [comment]
I tested the PR with the latest tarball and everything works as expected. The changes made by the PR were present.

**Pull request merged**